CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 0.86.0 / 2025-10-07
+
+> [!NOTE]
+> This release introduces the status subresource (behind the `StatusForConfigurationResources` feature gate) for `ServiceMonitor`, `PodMonitor` and `Scrapeconfig` custom resources. It is only supported for `Prometheus` resources.
+
+> [!IMPORTANT]
+> This release enables UTF-8 character support in label names, metric names and prometheus rule expressions for Prometheus 3.0+ compatibility.
```

Comment:
```suggestion
> This release enables automatic UTF-8 character support in label names, metric names and PrometheusRule expressions for Prometheus/PrometheusAgent resources running with version >= 3.0.0.
> To preserve backward compatibility, the admission webhook service validates PrometheusRule resources against the legacy Prometheus scheme by default (but it can be changed with the `--name-validation-scheme` flag).
```